### PR TITLE
Comment out ellipsis in code blocks

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Ellipsis in markdown code blocks are syntax errors.

In order to have smooth transition to Prettier and ESLint the PR comments out such bare ellipses. Refer the https://github.com/mdn/content/pull/18171#issuecomment-1179670484 for more context.

### CODE REVIEW

File: files/en-us/web/api/performance/getentries/index.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ None.
 
 ```js
 function use_PerformanceEntry_methods() {
-  console.log("PerformanceEntry tests ...");
+  console.log("PerformanceEntry tests…");
 
   if (performance.mark === undefined) {
-    console.log("... performance.mark Not supported");
+    console.error("performance.mark Not supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/getentries/index.md:49]:
```suggestion
    console.error("The property performance.mark is not supported");
```

File: files/en-us/web/api/performance/getentriesbyname/index.md [hunk 2]
```diff
@@ -47,10 +47,10 @@ A list of {{domxref("PerformanceEntry")}} objects that have the specified
 
 ```js
 function use_PerformanceEntry_methods() {
-  log("PerformanceEntry tests ...");
+  log("PerformanceEntry tests…");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/getentriesbyname/index.md:50]:
```suggestion
  console.log("PerformanceEntry tests…");
```

File: files/en-us/web/api/performance/getentriesbyname/index.md [hunk 3]
```diff
@@ -47,10 +47,10 @@ A list of {{domxref("PerformanceEntry")}} objects that have the specified
...
 
   if (performance.mark === undefined) {
-    log("... performance.mark Not supported");
+    log("performance.mark Not supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/getentriesbyname/index.md:53]:
```suggestion
    console.error("The property performance.mark is not supported.");
```

File: files/en-us/web/api/performance/getentriesbytype/index.md [hunk 4]
```diff
@@ -42,10 +42,10 @@ A list of {{domxref("PerformanceEntry")}} objects that have the specified
 
 ```js
 function usePerformanceEntryMethods() {
-  log("PerformanceEntry tests ...");
+  log("PerformanceEntry tests…");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/getentriesbytype/index.md:45]:
```suggestion
  console.log("PerformanceEntry tests…");
```

File: files/en-us/web/api/performance/getentriesbytype/index.md [hunk 5]
```diff
@@ -42,10 +42,10 @@ A list of {{domxref("PerformanceEntry")}} objects that have the specified
...
 
   if (performance.mark === undefined) {
-    log("... performance.mark Not supported");
+    log("performance.mark Not supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/getentriesbytype/index.md:48]:
```suggestion
    console.error("The property performance.mark is not supported.");
```

File: files/en-us/web/api/performance/setresourcetimingbuffersize/index.md [hunk 6]
```diff
@@ -48,10 +48,10 @@ function setResourceTimingBufferSize(maxSize) {
   }
   const supported = typeof performance.setResourceTimingBufferSize == "function";
   if (supported) {
-    log("... Performance.setResourceTimingBufferSize() = Yes");
+    log("… Performance.setResourceTimingBufferSize() = Yes");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/setresourcetimingbuffersize/index.md:51]:
```suggestion
    console.log("… Performance.setResourceTimingBufferSize() = Yes");
```

File: files/en-us/web/api/performance/setresourcetimingbuffersize/index.md [hunk 7]
```diff
@@ -48,10 +48,10 @@ function setResourceTimingBufferSize(maxSize) {
...
     performance.setResourceTimingBufferSize(maxSize);
   } else {
-    log("... Performance.setResourceTimingBufferSize() = NOT supported");
+    log("… Performance.setResourceTimingBufferSize() = NOT supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance/setresourcetimingbuffersize/index.md:54]:
```suggestion
    console.error("The method Performance.setResourceTimingBufferSize() is not supported.");
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 8]
```diff
@@ -32,7 +32,7 @@ function print_perf_entry(pe) {
 }
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:32]: Can you transform the `log(` on line 28 into a `console.log(`? Thank you!

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 9]
```diff
@@ -32,7 +32,7 @@ function print_perf_entry(pe) {
...
 function print_PerformanceEntries() {
   if (performance.mark === undefined) {
-    log("... performance.mark Not supported");
+    log("performance.mark Not supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:35]:
```suggestion
    console.error("The property performance.mark is not supported.");
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 10]
```diff
@@ -95,9 +95,9 @@ function print_PerformanceEntry(ev) {
       var supported = properties[j] in p[i];
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:95]: Please replace `log(` on line 92 into `console.log(`. Thank you!

```suggestion
      const supported = properties[j] in p[i];
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 11]
```diff
@@ -95,9 +95,9 @@ function print_PerformanceEntry(ev) {
...
       if (supported) {
         var pe = p[i];
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:97]:
```suggestion
        const pe = p[i];
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 12]
```diff
@@ -95,9 +95,9 @@ function print_PerformanceEntry(ev) {
...
-        log("... " + properties[j] + " = " + pe[properties[j]]);
+        log("… " + properties[j] + " = " + pe[properties[j]]);
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:98]:
```suggestion
        console.log("… " + properties[j] + " = " + pe[properties[j]]);
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 13]
```diff
@@ -95,9 +95,9 @@ function print_PerformanceEntry(ev) {
...
       } else {
-        log("... " + properties[j] + " = Not supported");
+        log("… " + properties[j] + " = Not supported");
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:100]:
```suggestion
        console.log("… " + properties[j] + " = Not supported");
```

File: files/en-us/web/api/performanceresourcetiming/domainlookupend/index.md [hunk 14]
```diff
@@ -56,9 +56,9 @@ function print_start_and_end_properties(perfEntry) {
     var supported = properties[i] in perfEntry;
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/domainlookupend/index.md:56]: On line 54, `var`-> `let`.
```suggestion
    const supported = properties[i] in perfEntry;
```

File: files/en-us/web/api/performanceresourcetiming/domainlookupstart/index.md [hunk 15]
```diff
@@ -51,9 +51,9 @@ function print_start_and_end_properties(perfEntry) {
     var supported = properties[i] in perfEntry;
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/domainlookupstart/index.md:51]: On line 49, `var`-> `let`.
```suggestion
    const supported = properties[i] in perfEntry;
```

File: files/en-us/web/api/performanceresourcetiming/domainlookupstart/index.md [hunk 16]
```diff
@@ -51,9 +51,9 @@ function print_start_and_end_properties(perfEntry) {
...
     if (supported) {
       var value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/domainlookupstart/index.md:53]:
```suggestion
    const value = perfEntry[properties[i]];
    if (supported) {
```

File: files/en-us/web/api/performanceresourcetiming/domainlookupend/index.md [hunk 17]
```diff
@@ -56,9 +56,9 @@ function print_start_and_end_properties(perfEntry) {
...
     if (supported) {
       var value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/domainlookupend/index.md:58]:
```suggestion
    const value = perfEntry[properties[i]];
    if (supported) {
```

File: files/en-us/web/api/performanceresourcetiming/connectend/index.md [hunk 18]
```diff
@@ -52,9 +52,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/connectend/index.md:54]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/connectstart/index.md [hunk 19]
```diff
@@ -50,9 +50,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/connectstart/index.md:52]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/redirectstart/index.md [hunk 20]
```diff
@@ -55,9 +55,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/redirectstart/index.md:57]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/requeststart/index.md [hunk 21]
```diff
@@ -54,9 +54,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/requeststart/index.md:56]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/responseend/index.md [hunk 22]
```diff
@@ -52,9 +52,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/responseend/index.md:54]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/responsestart/index.md [hunk 23]
```diff
@@ -50,9 +50,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/responsestart/index.md:52]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/secureconnectionstart/index.md [hunk 24]
```diff
@@ -53,9 +53,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/secureconnectionstart/index.md:55]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/workerstart/index.md [hunk 25]
```diff
@@ -55,9 +55,9 @@ function print_start_and_end_properties(perfEntry) {
     // check each property
     if (properties[i] in perfEntry) {
       const value = perfEntry[properties[i]];
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/workerstart/index.md:57]:
```suggestion
    const value = perfEntry[properties[i]];
    if (properties[i] in perfEntry) {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Found a few errors left fro `var` -> `const`. Also let's get rid of the undefined `log()` function, in favour of `console.log()`. (`log()` is a good idea in production code, but makes the snippet impossible to cut&paste)

**Comment by OnkarRuikar** [comment]
Thanks!
I am trying not to touch code as it may break some samples and lead to more PRs.

> Found a few errors left fro var -> const.

Yes I used to find some in those PRs(after merge). But I've stopped doing that…

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**